Src/Plugins/Security/PFXCryptographicKeyFileValidator.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
+{
+    internal static class PFXCryptographicKeyFileValidator
+    {
+#pragma warning disable IDE0060 // Remove unused parameter
+        public static string IsValid(
+            ref string matchedPattern,
+            ref Dictionary<string, string> groups,
+            ref bool performDynamicValidation,
+            ref string failureLevel,
+            ref string fingerprint)
+        {
+#pragma warning restore IDE0060
+            fingerprint = $"[cert={matchedPattern}]";
+
+            // This plugin does not perform any dynamic validation.
```

Comment:
>This plugin does not perform any dynamic validation [](start = 15, length = 51)

This isn't true! :) remove the comment and change line 28 to return true. Make sure not to proceed on this code path if the input value is false, though.

Reply:
if that's the case...should we change the behavior for the public azure devops validator? right now, it is the same thing.

---
In reply to: [554470513](https://github.com/microsoft/sarif-pattern-matcher/pull/96#discussion_r554470513) [](ancestors = 554470513)